### DISCUSSION TITLE
kaspersky malicious activity warning during essential tweaks install

### DISCUSSION THREAD

**Discussion by brutherb** [teeotsa/windows-11-debloat#26] [General]
I received a warning from Kaspersky during the essential tweaks install. Has anyone else encountered this. I can bypass the security alert but wondering if there is any validity to it.

**Comment by teeotsa** [comment]
Yeah, should be normal. Probs from downloading OOSHUTUP10 or OOSHUTUP10's config.

**Reply by brutherb** [reply]
thank you
